activerecord/test/cases/adapters/mysql2/datetime_precision_quoting_test.rb

```diff
@@ -25,28 +25,25 @@ class Mysql2DatetimePrecisionQuotingTest < ActiveRecord::Mysql2TestCase
     end
   end
 
-  test "no microsecond precision for MariaDB lt 5.3.0" do
-    stub_version "5.2.9-MariaDB" do
-      assert_no_microsecond_precision
-    end
-  end
-
   private
     def assert_microsecond_precision
       assert_match_quoted_microsecond_datetime(/\.000001\z/)
     end
 
     def assert_no_microsecond_precision
-      assert_match_quoted_microsecond_datetime(/\d\z/)
```

Comment:
`/\d\z/` is match to both datetime with precision and datetime without precision
```
[1] pry(main)> "2020-05-10 10:38:13".match?(/\d\z/)
=> true
[2] pry(main)> "2020-05-10 10:38:13.009789".match?(/\d\z/)
=> true
```

Reply:
👍 How about change the time to like `Time.now.change(sec: 12, usec: 3)` and change the both regex to more descriptive?

Reply:
Changed and updated commit message!